experiments/src/lib.rs

```diff
@@ -10,30 +10,52 @@ mod matcher;
 mod persistence;
 
 use anyhow::{anyhow, Result};
+pub use matcher::AppContext;
 use serde_derive::*;
-
+use std::path::Path;
+use uuid::Uuid;
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct Experiment {
     slug: String,
 }
+
+/// Optional custom configuration
+/// Could be used for debugging, or to change the server the SDK will communicate with
+/// Also includes an optional `uuid` for debugging/testing purposes
+/// The `uuid` would typically be retrieved from persisted storage
+pub struct Config {
```

Comment:
Can we move this in its own file as well? (just in case config grows and we can document more of its properties)